mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -465,6 +465,36 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib LDA.run()
+   */
+  def trainLDAModel(
+      data: JavaRDD[LabeledPoint],
```

Comment:
Thank you for letting me know about converting those types with `_convert_to_vector`. 
And sorry about that our discussion is here and there on github :( Could you please read my survey?
https://github.com/yu-iskw/spark/pull/3#issuecomment-113790325

I still wonder how we should treat `Long` in Java from python.